api/app/src/Service/UserService.php

```diff
@@ -39,6 +39,15 @@ public function addUser(User $loggedInUser, User $userToAdd, $data)
 
         $userToAdd->recreateRegistrationToken();
         $userToAdd->setCreatedBy($loggedInUser);
+
+        if ($loggedInUser->isLayDeputy()) {
```

Comment:
You can clean this up a bit more using a `match` - https://www.php.net/manual/en/control-structures.match.php